cockatrice/src/deck_loader.cpp

```diff
@@ -200,9 +200,19 @@ void DeckLoader::saveToStream_DeckZoneCards(QTextStream &out, const InnerDecklis
 QString DeckLoader::getCardZoneFromName(QString cardName, QString currentZoneName)
 {
     CardInfo *card = db->getCard(cardName);
-    if(card && card->getIsToken())
+    if (card && card->getIsToken())
         return DECK_ZONE_TOKENS;
 
     return currentZoneName;
 }
 
+QString DeckLoader::getCompleteCardName(QString cardName) const
```

Comment:
The passed in `QString cardName` can be made `const QString cardName`. This ensures that the passed in param is not mutated. 
The trailing method `const` only states this method will not mutate class state.